src/pre_commit_vauxoo/pre_commit_vauxoo.py

```diff
@@ -45,6 +47,21 @@ def get_files(path):
     return ls_files
 
 
+def get_uninstallable_modules(src_path) -> set:
+    """Find all odoo modules that are set as not installable. They must have a key 'installable' with a False value
+    in order to be considered not installable.
+
+    :return: A set of strings, each one representing the relative path (from repo dir) to an uninstallable module.
+    """
+    results = set()
+    for path in glob.glob(os.path.join(src_path, "**/__manifest__.py"), recursive=True):
```

Comment:
The odoo modules are only in the main repo path

 - `repo/module1/__manifest__.py`

So, it could be enough:

```suggestion
    for path in glob.glob(os.path.join(src_path, "*/__manifest__.py")):
```

Reply:
Test repo is inside `resources` (`repo_dir/resources/modulesX`), it does not find any manifest with your suggestion but it would work with every day to day repos. We could fix that and copy paths directly to `repo_dir`, although I think recursive does not hurt anything and saves us in weird non-standard cases.

Reply:
AFAIK Odoo supports modules into deeper directories (see Odoo.sh), so maybe it could be a good idea to support it also here.

Reply:
OMG you are right!

We need to move the unittest to use a standard repo structure

I prefer don't use `recursive=True` since that for big projects it could spend more time than needed since that we don't have any project using a module into another module, even it should be checked from other lint but it will force to ignore it

So, better only ignore the very specific know standard cases

and fix the unittest path repo

Reply:
odoo-sh support `git submodules`

It looks like modules into deeper directories but they are not since that they have a particular project separated with normal structure

Reply:
Check our guidelines related to structure directory:
 - https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#complete-structure

`addons/<my_module_name>/` = `REPO/<my_module_name>/`

You are talking about:
 - `addons/<my_module_name1>/<my_module_name2>/<my_module_name3>/`

It is not supported in our guidelines and even for our current lints

Check MQT get_modules method, vxci, lints and so on

Even if you like support this new way you need to start to proposing it in the guidelines first then create a new PR here, MQT, vxci and so on

Reply:
I understand, thanks.

Reply:
@moylop260 Do I change the test structure on another PR so we can then merge this one?

Reply:
Let's to do the same PR but extra commit, please